ui/src/components/ripple-out/support-sections/RippleOutGiveSection.js

```diff
@@ -0,0 +1,223 @@
+import {
+  Flex,
+  Box,
+  Grid,
+  GridItem,
+  Center,
+  Text,
+  Spacer,
+  VStack,
+  Image,
+  HStack,
+  Link,
+} from '@chakra-ui/react';
+import RippleOutBarChart from './RippleOutBarChart';
+import RippleOutGivingCard from './RippleOutGivingCard';
+import {
+  subHeaderFontSize,
+  headerFontSize,
+  bodyFontSize,
+} from '../RippleOutTextStyle';
+import { useState } from 'react';
+import RippleOutCalculatorSection from './RippleOutCalculatorSection';
+
+const RippleOutGiveSection = ({ giveData }) => {
+  const data = giveData.content;
+  const [slider, setSlider] = useState(0);
+
+  return (
+    <>
+      <Text
+        fontSize={subHeaderFontSize}
+        textStyle="darker_grotesque_black"
+        lineHeight="0.9em"
+      >
+        GIVE TO THE CAMPAIGN
+      </Text>
+      <Box
+        bg="#ffffff"
+        px="10"
+        py="5"
+        border="2px solid #182E57"
+        borderRadius="2xl"
+        boxShadow="lg"
+      >
+        <Grid templateColumns={['repeat(1, 2fr)', 'repeat(2, 1fr)']}>
+          <GridItem>
+            <Center h="100%">
+              <Flex flexDir="column">
+                <Text
+                  color="#96825B"
+                  textStyle="darker_grotesque_bold"
+                  lineHeight={['1em', '0.5em']}
+                  fontSize={['2xl', '3xl']}
+                >
+                  Fundraising Target
+                </Text>
+                <Text
+                  fontSize={headerFontSize}
+                  textStyle="darker_grotesque_black"
+                  lineHeight="1em"
+                >
+                  HKD $2MILLION
+                </Text>
+                <Text fontSize={bodyFontSize} lineHeight="1.3em">
+                  raised by end of March 2024
+                </Text>
+                <Text mt={5} fontSize="lg" textStyle="darker_grotesque_bold">
+                  {data[0].data}
+                </Text>
+              </Flex>
+            </Center>
+          </GridItem>
+          <GridItem display="flex" justifyContent="flex-end" minH="40vh">
+            <RippleOutBarChart height="100%" width={['100%', '80%']} />
+          </GridItem>
+        </Grid>
+      </Box>
+      <Text fontSize={bodyFontSize} fontWeight={500}>
+        {data[2].data[0]}
+        <Box as="span" fontWeight={700}>
+          {data[2].data[1]}
+        </Box>
+        {data[2].data[2]}
+      </Text>
+
+      {/* Plans ahead */}
+      <Text
+        fontSize={subHeaderFontSize}
+        textStyle="darker_grotesque_black"
+        lineHeight="0.9em"
+      >
+        PLAN AHEAD
+      </Text>
+      <Flex>
+        <VStack>
+          <Text fontSize={bodyFontSize} fontWeight={500}>
+            Here’s an illustration of how a small act of giving over time can go
+            a long way to contribute to the{' '}
+            <Box as="span" fontWeight={700}>
+              Ripple Out Campaign
+            </Box>
+          </Text>
+          <Image
```

Comment:
Need a swipe interaction for this according to design. atm the only way for people to switch between slides is to click dots which is not very intuitive